plugins/tiddlywiki/tiddlyweb/tiddlywebadaptor.js

```diff
@@ -102,6 +103,53 @@ TiddlyWebAdaptor.prototype.getStatus = function(callback) {
 	});
 };
 
+TiddlyWebAdaptor.prototype.startSSE = function() {
+	// Get the mount point in case a path prefix is used
+	var host = this.getHost();
+	// Make sure it ends with a slash (it usually does)
+	if(host[host.length - 1] !== "/") {
+		host += "/";
+	}
+	// Setup the event listener
+	this.setupSSE(host);
+};
+function debounce(callback) {
+	var timeout = null;
+	return function() {
+		clearTimeout(timeout);
+		timeout = setTimeout(callback,$tw.syncer.throttleInterval);
+	};
+}
+
+TiddlyWebAdaptor.prototype.setupSSE = function (host, refresh) {
+	if (window.EventSource) {
+		var self = this;
+		if(this.eventsource && this.eventsource.readyState !== this.eventsource.CLOSED)
+			this.eventsource.close();
```

Comment:
It should look like this: 

```
if(this.eventsource && this.eventsource.readyState !== this.eventsource.CLOSED) {
		this.eventsource.close();
	}
```

Otherwise it's super error prone if the source will ever be changed here.